docs/run_publish/references.md

```diff
@@ -100,6 +105,42 @@ This displays the current version.
 0.19.1
 ```
 
+### reindex
+
+- In order to use this command you need to have `@subql/node` v1.10.0 or above.
+
+When using reindex command, historical must be enabled for the targeted project(`--disable-historical=false`).
```

Comment:
Is this set to false by default (i.e. is historical enabled by default)?
Is there another way they can check? E.g printed on project start

Reply:
when starting the project, it should log if historical is enable or not

Reply:
@bz888 please add a sentence about how to check if this is enabled or not